packages/appinio_swiper/lib/appinio_swiper.dart

```diff
@@ -1,679 +1,743 @@
+  /// Sets whether the card should angle in the opposite direction when it is
+  /// dragged from the bottom half.
+  ///
+  /// Defaults to true.
+  final bool invertAngleOnBottomDrag;
+
+  /// What swipe directions to allow.
+  ///
+  /// Swipes triggered by a controller are always allowed and are not affected
+  /// by [swipeOptions].
+  final SwipeOptions swipeOptions;
+
+  /// The minimum distance a user has to pan the card before triggering a swipe
+  /// animation.
+  ///
+  /// If a pan is less than [threshold] the card will animate back to it's
+  /// original position and the card stack will not change.
+  final double threshold;
+
+  /// Set to true to disable swiping.
   final bool isDisabled;
 
-  /// set to false if unswipe should be disabled
-  final bool allowUnswipe;
-
-  /// set to true if you want to loop the items
+  /// Set to true if you want the card stack to loop back to the beginning when
+  /// the user swipes to the end.
+  ///
+  /// The looped cards will be displayed under the foreground card as if
   final bool loop;
 
-  /// set to true if the user can unswipe as many cards as possible
-  final bool unlimitedUnswipe;
-
-  /// function that gets called with the new index and detected swipe direction when the user swiped or swipe is triggered by controller
+  /// Callback that fires with the direction and new index whenever a swipe is
+  /// triggered by the user or the [controller].
   final OnSwipe? onSwipe;
 
-  /// function that gets called when there is no widget left to be swiped away
+  /// Function that is called when the card stack runs out of cards.
   final VoidCallback? onEnd;
 
-  /// function that gets triggered when the swiper is disabled
+  /// Function that is called when a user attempts to swipe but swiping is
+  /// disabled.
   final VoidCallback? onTapDisabled;
 
-  /// function that gets called with the boolean true when the last card gets unswiped and with the boolean false when there is no card to unswipe
+  /// Function that gets called when an unswipe is triggered.
+  ///
+  /// This is called with a boolean flag that is true if the unswipe was
+  /// successful and false if it was not (there was no swipe to undo).
   final OnUnSwipe? unswipe;
 
-  /// direction in which the card gets swiped when triggered by controller, default set to right
-  final AppinioSwiperDirection direction;
+  /// The default direction in which the card gets swiped when triggered by
+  /// controller.
+  ///
+  /// Defaults to [AxisDirection.right].
+  final AxisDirection defaultDirection;
 
   const AppinioSwiper({
     Key? key,
-    required this.cardsBuilder,
-    required this.cardsCount,
+    required this.cardBuilder,
+    required this.cardCount,
     this.controller,
-    this.padding = const EdgeInsets.symmetric(horizontal: 20, vertical: 25),
     this.duration = const Duration(milliseconds: 200),
-    this.maxAngle = 30,
+    this.maxAngle = 15,
```

Comment:
I halved the default max angle because I rewrote the angle calculation to be tied to the screen size which made it rotate faster. Before, you'd never get near the max angle while swiping on a small screen.